packages/build/src/barque.spec.ts

```diff
@@ -112,44 +112,44 @@ describe('Barque', () => {
 
   describe('.determineDistro', () => {
     it('determines distro for ubuntu', async() => {
-      const distro = barque.determineDistro('linux');
+      const distro = barque.determineDistro(BuildVariant.Linux);
```

Comment:
Can we make this simpler rather than more complex, and just make `type BuildVariant = 'linux' | 'debian" ...`, or, ideally, `type BuildVariant = 'tgz' | 'deb' | 'rpm' | 'msi' | ...`?

Reply:
Yeah, I think that's the point of the ticket -- try to actually be intentional about what we're differentiating here ... we can also replace the name "build variant" if you have a good idea for an alternative :)

Reply:
Mh okay - unfortunately it's not really replaceable directly. We're using the Build variant as also the system identifier to construct our tarball names. So we have `mongosh-v1.0.0-win32.zip` for example. In the end that change you commented is not making it more complex but more explicit. I changed the signature to just use the type of the enum.

Reply:
Hm yeah -- it's just confusing that neither `linux` nor `rhel` nor `debian` actually do what they stand for, they're just differentiating the package formats and not the OSes... I don't think it's bad to leave this as-is, but if we can move away from it in some way, we probably should